django/cantusdb_project/main_app/views/redirect.py

```diff
@@ -67,15 +67,42 @@ def redirect_indexer(request, pk: int) -> HttpResponse:
 
 def redirect_office(request) -> HttpResponse:
     """
-    Redirects from office/ (à la OldCantus) to offices/ (à la NewCantus)
+    Redirects from office/ (à la OldCantus) to services/ (à la NewCantus)
 
     Args:
         request
 
     Returns:
         HttpResponse
     """
-    return redirect("office-list")
+    return redirect("service-list")
```

Comment:
I know we aren't consistent with legacy redirects, but I think these should have `permanent = True`.